packages/docs/src/pages/en/features/icon-fonts.md

```diff
@@ -82,7 +82,7 @@ export default createVuetify({
 
 <alert type="info">
 
-  Note the `mdi:` prefix used in the *second* icon. It is necessary to provide the **correct** prefix when rendering icons that are **not** part of the default set.
+  It is not necessary to provide a prefix (such as `mdi:`) for icons from the default set
```

Comment:
```suggestion
  Icons from the default set can omit the library prefix; e.g. **icon="mdi:close"** → **icon="close"**.
```

Reply:
What about the current wording don't you like?